apps/language_server/lib/language_server/server.ex

```diff
@@ -789,4 +790,25 @@ defmodule ElixirLS.LanguageServer.Server do
   defp set_project_dir(state, _) do
     state
   end
+
+  defp create_gitignore(%{project_dir: project_dir} = state) do
+    with gitignore_path <- Path.join([project_dir, ".elixir_ls", ".gitignore"]),
+         false <- File.exists?(gitignore_path),
+         :ok <- gitignore_path |> Path.dirname() |> File.mkdir_p(),
+         :ok <- File.write(gitignore_path, "*", [:write]) do
+      state
+    else
+      true ->
+        JsonRpc.log_message(:warning, ".elixir_ls/.gitignore already exists")
```

Comment:
Since this is an expected case, we don't need this log message.

Reply:
@axelson fixed.